app/src/main/java/com/example/ubclaunchpad/launchnote/photoBrowser/AllPhotosFragment.kt

```diff
@@ -1,29 +1,64 @@
+        "https://lh3.googleusercontent.com/xt9VRukTua36-Y9n96d_mahtNWiPhzX1bAm4LjM0O8iBGUpgub2mS9MvfLAyncfn_EnuCJAU8evsUA7d0-WpieuoLrFTwUogLZlMUTSbd9N6S8Km0S-37Wu_TsNu6tGjFvtKwFyMChnl1GFKxtHaF_AzCKtRH6PHC0MpiSYfdK0387uqlP_DccssjWsv1jp_xTEvuNsfBrfv5lfw9Cd32CfUIPbFkKlMdmoOSnZx2e1hiyGasNaSgoPDI5Dd1OFpLJ30fVAZwMI1Rrn35aw51NCVhVOHLrsdkPTg8vO7mYQGLqHVP905wj9c5DuocgswPtXxLqdJug_NQrSHllUbvbT8CCTUI5uVfMtg9RaXs_PyJWbFeGpPA3vKjSJY-cCLoBm_dcC8y6WLDCKI1hObzu-T_MSiKwSg_9kPtrzBBHzS83vRguxIB8FMq8arIN67ofNippAdQZtOOoi8NbTHzBwS-8LTxBhSWI5_ATri6MzZQN0H6tOonniSwvHNUogDwvjpQDSzGX6vPEK7IGLNKQLTykeHdCX25bdlQsKx5Z6oCkp58P3dbHhcsTKZ3uxlTM0CurvOnt7VdP7LzvSqJUGcrOTnFluv0xMhy1gWug=w647-h924-no",
+        "https://lh3.googleusercontent.com/yF9QqXskZqMCneu5lJv5I-Qeip96eAt9coWuwwT7quz5WSG5gNN3IqXi1yKUL4FSRi96zBpbhBkdw2Qvz-NMePWF02RR2ygAy7StvzrgBbD2YVx8zYG1a9Ynhi5x20aa0cfYiEarbMjqLQPFHCRu3f0Qc3Sfwzm4D-A5qDPxkI671cElKm8UxZSYhP8W0zcmDIHt_-wCT6IkvAcOjNelRvBOOduYBFRq7FmmR-FnMjZ3Kg6ith1aJSN_B3wvztVaIw4tr8Exgqb8zJblsQjoFQNwjXoiB6E1ysHRnYVqi-g7w43j_nShOuwumA1jz2Tl_mJfBgtcAF9SacDTa-ajty912QqasA4nXUIGZqotxW5TLKhDUx_rEovsCFvSf1_skpfnhrKyLsL8GdtVbNUc7aTK2VEZbL2BStWKn1gCGmRbZWCPATZufqg1nCXOYm-7G7qtmIST5O4oPM52iKUhneLriu8VJnUVhnZBcId7BqV1QdJybXUi1kHVc7Sl0yNaJ39NsFnhUjIL7vcvi8Nz0QLCZlzoWgEL62WGrV9UQy0DFlsU4dh73bAaOMxTBUxlbdrcJrhuUJwKRjdsljMLy2tkKkVbBdo2WxJWWqAZuQ=w798-h923-no",
+        "https://lh3.googleusercontent.com/qVIQyBzvkn6UPtWMVzfD3zIxcNjQq6_xnw1-pAbo9qxbSe-if_7376eT2FWUJWoXVhVVXuTLIb3dRvSP4J37Z1LJmrd1TV0ItKwUbBJUzyuEs1Fx4Ec-qjWVCv5Sj4ELChLCOnGzSmbXMz_duQmgz-iQfLtD2NKWcyUBDijZOSp16owkxr04a6Uc4N-xSzRTq4C-xZi-iFvksdGjJuBFLvScjNkt3DCWQigUF9frzEzneB5tmjhD2YfSi9SPFg4HA5_ZDBL7jh5XXOLlcb6bfCebA55V1YX_IXMdcWMe5-TcEjMtHPAFaqx69jPwgLxiQ_yAYkK7G_lQ-fzIYq3oqOZLJQkxxfp7n--zqpYyrlLWfa-zRcrtIT1fH3KOczwDpnBX-DcyK6ygptoHW1JiRAxHFW3gD2aBBNjcSoJpM6y8xfNNrM3MPlRNeu_2Sl5cnZ-okSlJ0K9eEG3zyHQADk_LaGWUUmq5q1zkDSu1bhquynEu8hQ54kkNYn6GBtyisDRNMcV6MD-aZlmQgj8zVTIyfj7Rb_43D84RF1hYeVhdeR9T5veAH9ETHgcrt5t7ALi9wX4N1vfiVf1FdB2zfRpHArgJ7anzcv11GfixOQ=w693-h923-no"
+    )
+
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
         return inflater.inflate(R.layout.fragment_all_photos, null)
     }
 
     override fun onResume() {
         super.onResume()
-        loadImages()
+        if (resources.configuration.orientation == Configuration.ORIENTATION_PORTRAIT) {
+            initViews(3) // 3 columns while straight
```

Comment:
put 3 into a "NUM_COLUMNS_PORTRAIT" constant instead, in case we'll want to use it again in the future. Generally in Android you don't want unlabelled ints and strings being used.